fontforgeexe/fontinfo.c

```diff
@@ -2657,7 +2657,7 @@ static int CheckNames(struct gfi_data *d) {
     buts[0] = _("_OK"); buts[1] = _("_Cancel"); buts[2]=NULL;
 
     if ( u_strlen(ufont)>63 ) {
-	ff_post_error(_("Bad Font Name"),_("A PostScript name should be ASCII\nand must not contain (){}[]<>%%/ or space\nand must be shorter than 63 characters"));
+	ff_post_error(_("Bad Font Name"),_("A Fontname should be ASCII\nand must not contain (){}[]<>%%/ or space\nand must be shorter than 63 characters"));
```

Comment:
I hadn't noticed the title - the error should always say the same as the title. Anything else is bad UX. (And the title still isn't 100% = label name).

I'm no code expert but I'm fairly sure it only responds to "Bad Font Name" so there's no reason for not using the precise "Fontname" 

(the "A" should be removed, yes)